### DISCUSSION TITLE
Weirdness with Auckland's City value in New Zealand

### DISCUSSION THREAD

**Discussion by uqlwil35** [komoot/photon#684] [Q&A]
Hi there,

I'm just wanting to open a discussion around how the City value is populated in search results. 

My specific problem is that when I search for 'Auckland' which is a city in New Zealand I get the following results:

`https://photon.komoot.io/api/?q=Auckland&limit=1`
```json
{
  "features": [
    {
      "geometry": {
        "coordinates": [
          174.7631803,
          -36.852095
        ],
        "type": "Point"
      },
      "type": "Feature",
      "properties": {
        "osm_type": "N",
        "osm_id": 292806332,
        "country": "New Zealand",
        "osm_key": "place",
        "city": "Waitematā",
        "countrycode": "NZ",
        "osm_value": "city",
        "postcode": "1010",
        "name": "Auckland",
        "state": "Auckland",
        "type": "district"
      }
    }
  ],
  "type": "FeatureCollection"
}
```

What is interesting is that it specifies the city of Auckland is Waitematā. However, when I run the same search in Nominatim, I get this:

`https://nominatim.openstreetmap.org/search?format=json&addressdetails=1&q=Auckland&limit=1`
```json
[
  {
    "place_id": 856074,
    "licence": "Data © OpenStreetMap contributors, ODbL 1.0. https://osm.org/copyright",
    "osm_type": "node",
    "osm_id": 292806332,
    "boundingbox": [
      "-37.012095",
      "-36.692095",
      "174.6031803",
      "174.9231803"
    ],
    "lat": "-36.852095",
    "lon": "174.7631803",
    "display_name": "Auckland, Waitematā, Auckland, 1010, New Zealand",
    "class": "place",
    "type": "city",
    "importance": 0.7584956132983275,
    "icon": "https://nominatim.openstreetmap.org/ui/mapicons/poi_place_city.p.20.png",
    "address": {
      "city": "Auckland",
      "state": "Auckland",
      "ISO3166-2-lvl4": "NZ-AUK",
      "postcode": "1010",
      "country": "New Zealand",
      "country_code": "nz"
    }
  }
]
```

In this case the city (under address) is clearly Auckland, and the only mention of Waitematā is within the display_name.

**Comment by lonvia** [comment]
There were a couple of bugs in the Nominatim update mechanism which caused the bad city results. They have been fixed a while ago but needed a database reimport to make the bad data go away. The database behind the photon exports is now all new and uses the latest Nominatim 4.1. The Auckland issue is gone.

Note that the servers behind nominatim.openstreetmap.org are completely independent from the one we use to create the photon extracts (which are in turn used by photon.komoot.io). They tend to even run slightly different versions of Nominatim. So differences are to be expected.
